### PR TITLE
Updated JDK package installation into one dependency

### PR DESCRIPTION
**Pull request by YeshwanthReddySangam**
Combined JavaDevelopmentKitInstallation and DependencyPackageInstallation into one single dependency

### CODE REVIEW

**Comment by YeshwanthReddySangam** [comment]
@microsoft-github-policy-service agree

File: website/docs/dependencies/0050-install-java-jdk.md [hunk 1]
```diff
@@ -32,31 +32,27 @@ The following section describes the parameters used by the individual component
 
 | **Parameter** | **Required** | **Description**                                                                                                 |
 |---------------|--------------|-----------------------------------------------------------------------------------------------------------------|
```
Comment (yangpanMS) [inline website/docs/dependencies/0050-install-java-jdk.md:34]: Thanks for updating the document. Please add 1/2 sentences about how this derives from the DependencyPackageInstallation

**Review by yangpanMS — APPROVED** [review]
We don't really have this scenario yet but this now forbid users from using their own packaged JDK. It has to be downloaded. Can we find a path where if the blob is not defined, the JdkInstallation proceeds to find the package via package name?

File: src/VirtualClient/VirtualClient.Dependencies/JDKPackageDependencyInstallation.cs [hunk 2]
```diff
@@ -45,6 +46,8 @@ public JavaDevelopmentKitInstallation(IServiceCollection dependencies, IDictiona
         /// </summary>
         protected override async Task ExecuteAsync(EventContext telemetryContext, CancellationToken cancellationToken)
         {
+            // Install JDK package
+            await base.ExecuteAsync(telemetryContext, cancellationToken);
```
[the remark this answers is not in the document]
Comment (YeshwanthReddySangam) [reply src/VirtualClient/VirtualClient.Dependencies/JDKPackageDependencyInstallation.cs]: Updated documentation for transition between versions

File: website/docs/dependencies/0050-install-java-jdk.md (hunk 1 above)
Comment (YeshwanthReddySangam) [reply website/docs/dependencies/0050-install-java-jdk.md:40]: Updated documentation along with installation steps

**Review by yangpanMS — APPROVED** [review]
Great work, thanks!

**Pull request merged**